### ISSUE TITLE
[FEATURE] SponsoreBlock Integration?

### ISSUE THREAD

**Issue by mridiot0** [toasterofbread/spmp#116]
Hey, just wanted to ask if there's any plan for integrating [Sponsorblock](https://sponsor.ajay.app/)  In this application.
I know YouTube music itself doesn't contain anything non music but the music that are from YouTube contains segments like channel intro/outro, non music segments. It would be very nice if you could integrate that ☺️. 
Thank You.

**Comment by Yash5320** [comment]
Ya this one

**Comment by Rafee-M** [comment]
SimpMusic has SponsorBlock integration if needed for reference

**Comment by mridiot0** [comment]
Yeah! I've used that one but in my device it's a bit laggay and stops in the middle of a session. Thanks for the reference though.
